src/WinRPM.jl

```diff
@@ -324,12 +324,12 @@ type RPMVersionNumber
     s::AbstractString
 end
 Base.convert(::Type{RPMVersionNumber}, s::AbstractString) = RPMVersionNumber(s)
-Base.(:(<))(a::RPMVersionNumber,b::RPMVersionNumber) = false
-Base.(:(==))(a::RPMVersionNumber,b::RPMVersionNumber) = true
-Base.(:(<=))(a::RPMVersionNumber,b::RPMVersionNumber) = (a==b)||(a<b)
-Base.(:(>))(a::RPMVersionNumber,b::RPMVersionNumber) = !(a<=b)
-Base.(:(>=))(a::RPMVersionNumber,b::RPMVersionNumber) = !(a<b)
-Base.(:(!=))(a::RPMVersionNumber,b::RPMVersionNumber) = !(a==b)
+@compat Base.:(<)(a::RPMVersionNumber,b::RPMVersionNumber) = false
```

Comment:
0.7.15  and WinRPM requires 0.7.20 so it's all good. On a side note I'm thinking about changing to Libz instead of ZLib, it seams Libz is used only once on line 158